### ISSUE TITLE
Style related - Cannot apply update. Need to do a full reload!

### ISSUE THREAD

**Issue by Borvik** [pmmmwh/react-refresh-webpack-plugin#314]
So, I've been trying to get this refresh working, and I've been partially successful.

It seems if I try importing a stylesheet (I've tried `css` and `scss`) it doesn't work properly

![image](https://user-images.githubusercontent.com/1764750/108609015-f0f8ea00-7398-11eb-856c-03d28715609a.png)

At first glance it might seem like #182 except that the file changing is _not_ the stylesheet.

If however, I remove the stylesheet import and save it, do a full reload then make a simple verbiage change inside the same component, it works just fine.

![image](https://user-images.githubusercontent.com/1764750/108609114-b6438180-7399-11eb-8e0f-b6bc9844beca.png)

I am using `style-loader`, and removed `browserlist`.

I did use the `typescript-without-babel` example (which works OOTB), as a template for how to integrate.

For full detail a weird thing I'm noticing (which may not be related to the refresh) is that on the `typescript-without-babel` example when it starts up I see these 3 logs:

```
[HMR] Waiting for update signal from WDS...
[WDS] Hot Module Replacement enabled.
[WDS] Live Reloading enabled.
```

At one point mine did have all 3, but now just has the first one - though even though it doesn't say enabled it basically works (except for the style issue described above).

I do have a repo that should be able to reproduce this: https://github.com/Borvik/react-component-library

Currently when you run it changes to the `App` component will trigger a full reload - you can see it switch to refresh working, by simply commenting out the style import in `src/demo/app.tsx` saving, and then altering the text on line 11 of the same file.

Any thoughts?

**Comment by pmmmwh** [comment]
Hi, can you test the `0.5.0` beta and see if it works for you? There are some fixes related to this issue that is only in that beta.

**Comment by Borvik** [comment]
Tested, issue is still occurring.

Basic test scenario - click provided button so state has changed, then make change to file `app.tsx`.

When no style is imported - component refreshes fine and preserves state.
When style is imported - full page refresh, state is not preserved.

**Comment by pmmmwh** [comment]
Fixed in `0.5.0-beta.1`.

**Comment by Borvik** [comment]
Confirmed! Thanks.
